fobi_custom/plugins/form_elements/fields/utils.py

```diff
@@ -4,3 +4,13 @@ def choices_to_help_text(choices):
         help_text += '- ' + choice + '<br />'
 
     return help_text
+
+
+def choices_to_help_text_skip_first(choices):
```

Comment:
Another way to implement this could have been to add a `skip_first` keyword argument to the parameter list of `choices_to_help_text` and set it to default to `False`, e.g.:

```python
def choices_to_help_text(choices, skip_first=False):
    ...
```

That way all of the code that formats choices for display would share the same function, and if you ever wanted to change the input or output format you'd only have to adjust one function.

Reply:
Much more elegant 😎